components/SiteNavbar.js

```diff
@@ -1,10 +1,13 @@
-import Navbar from "react-bootstrap/Navbar";
-import Nav from "react-bootstrap/Nav";
+import { Nav, Navbar } from "react-bootstrap";
+import { useRouter } from 'next/router'
 import Link from "next/link";
 
 export default function SiteNavbar() {
+  const router = useRouter();
+  const pathName = router.asPath;
```

Comment:
Passing this into `activeKey` fixes a bug with the active nav button functionality of react-bootstrap

Reply:
Awesome! Thanks for finding a fix for this I was stumped for so long.